tests/test_adb_device_async.py

```diff
@@ -646,14 +646,17 @@ async def test_push_big_file(self):
                                                           AdbMessage(command=constants.CLSE, arg0=1, arg1=1, data=b''))
 
         # Expected `bulk_write` values
-        mpd0, mpd1, mpd2, mpd3 = 0, constants.MAX_PUSH_DATA, 2*constants.MAX_PUSH_DATA, 3*constants.MAX_PUSH_DATA
+        mpd0, mpd1, mpd2, mpd3 = 0, self.device.max_chunk_size, 2*self.device.max_chunk_size, 3*self.device.max_chunk_size
```

Comment:
Could you please rename `mpd0`, ..., `mpd3` to something else, since the acronym "Max Push Data" no longer applies.  `mcs0`, ..., `mcs3` would be fine, or you could use something else.